languages/swift/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,37 @@
+You are an elf working in Santa Claus' logistics division and you have been given a pair of tasks from the boss for upgrading the system's software.
```

Comment:
I missed the debate about Santa Claus (I understand, that it might not be familiar to all students) - and maybe I'm not aware of some common agreement regarding this topic. 

My personal thoughts by reading these 2 exercises: It is perfectly doable without knowing Santa Claus ... it is then just "some business". :)

Reply:
I quite like the Santa Claus theme :) It gives it a bit of flourish. 

> I missed the debate about Santa Claus

What was the debate?

Reply:
@iHiD Are you aware of any issues with using Santa Claus as the story?

Reply:
Nein. We're good.